src/unstable/components/Form/index.tsx

```diff
@@ -71,11 +71,20 @@ export default class FormHOC extends React.Component<
 		super(props);
 
 		this.state = {
-			value: this.props.value || {},
+			value: this.props.value || FormHOC.defaultValueForSchema(props.schema),
+
 			schema: this.parseSchema(this.props.schema),
 		};
 	}
 
+	static defaultValueForSchema(schema: JSONSchema6) {
```

Comment:
Generally speaking, it is bad practice to expose static methods from React components. You can either have this as a utility function in some `utils` file, or, honestly speaking, you can just duplicate it in both places, as `JellyForm` shouldn't be aware of its child `Form` IMO. Also, you can rewrite it as 
```
	static defaultValueForSchema(schema: JSONSchema6) {
   return schema.type === 'array' ? [] : {}
	}
```